src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
According to the task description: To test the program, in main() method, create three balls using getRandomBall() method from Lottery class. And print information about them in the console.
Please, read task carefully and implement the logic